lib/models/project.js

```diff
@@ -104,6 +104,10 @@ Project.prototype.hasDependencies = function() {
  */
 Project.prototype.setupNodeModulesPath = function() {
   this.nodeModulesPath = nodeModulesPath(this.root);
+
+  this.emberCliNodeModulesPath = this.nodeModulesPath &&
```

Comment:
Ah, okay, will do that. Wasn't really feeling good about resolving the path that way anyhow.